interfaces/Portalicious/src/app/pages/project-registrations/components/export-registrations/export-registrations.component.ts

```diff
@@ -43,6 +47,7 @@ import { ToastService } from '~/services/toast.service';
     DatePickerModule,
     FloatLabelModule,
     FormsModule,
```

Comment:
```suggestion
```

Maybe we can/should get rid of this now? Not sure tbh - needs to be tested. Same comment applies to other components that have the same change of adding `ReactiveFormsModule`